.github/labeler.yml

```diff
@@ -11,7 +11,7 @@ dependencies:
   - mockserver/package.json
   - mockserver/yarn.lock
   - web/package.json
-  - web/yarn.lock
+  - web/pnpm-lock.yaml
```

Comment:
More like this I think:

<img width="333" alt="Screenshot 2023-01-30 at 22 38 47" src="https://user-images.githubusercontent.com/3296643/215601276-acefad54-3e81-465e-a70e-299aed890d2a.png">